.circleci/config.yml

```diff
@@ -81,27 +81,21 @@ commands:
   setup_tox:
     description: "Install tox"
     steps:
+      - restore_pip_cache
       - run: pip install tox
 
-  restore_tox_cache:
-    description: "Restore .tox directory from previous runs for faster installs"
+  restore_pip_cache:
+    description: "Restore pip download cache"
     steps:
       - restore_cache:
-          # In the cache key:
-          #   - .Environment.CIRCLE_JOB: We do separate tox environments by job name, so caching and restoring is
-          #                              much faster.
-          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}
+          key: pip-cache-{{ checksum "tox.ini" }}-{{ checksum "setup.py" }}
 
-  save_tox_cache:
-    description: "Save .tox directory into cache for faster installs next time"
+  save_pip_cache:
+    description: "Save pip download cache"
     steps:
       - save_cache:
-          # In the cache key:
-          #   - .Environment.CIRCLE_JOB: We do separate tox environments by job name, so caching and restoring is
-          #                              much faster.
-          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}
-          paths:
-            - ".tox"
+          key: pip-cache-{{ checksum "tox.ini" }}-{{ checksum "setup.py" }}
+          paths: /root/.cache/pip
```

Comment:
will this impact versions installed with `tox`? will `pip` try to serve stale versions if they are in the cache? (e.g. `pip install flask` vs `pip install flask>=1.0.0` vs `pip install flask==1.0.0`)

Reply:
No, pip will still ask PyPI index for the version to install. It won't just download it again.

Reply:
got it, I thought I had problems with it in the past where if a local cached version met the constraints it'll just use it, e.g. if you have `flask==1.1.0` downloaded and ask for `flask>=1.1.0,<1.2.0`, even if `1.1.1` is on PyPI it'll use the cached version.

maybe I am misremembering/being overly cautious here.